tigerpath/majors_and_certificates/majors/CLA_2018.json

```
@@ -0,0 +1,373 @@
+              "course_list": [
+                "CLA *"
+              ]
+            },
+            {
+              "name": "Ancient Greek or Latin",
+              "max_counted": 1,
+              "min_needed": 1,
+              "explanation": "Each student must successfully complete either Ancient Greek or Latin to the level of 108 or achieve an equivalent level of knowledge, as demonstrated through test scores (SAT, AP), a placement exam, or coursework (including through a summer language program). However, at least one language course must be taken at Princeton.",
+              "course_list": [
+                "LAT 108",
+                "CLG 108"
+              ]
+            },
+            {
+              "name": "Departmentals",
+              "max_counted": null,
+              "min_needed": null,
+              "explanation": "<i>Editor's Note: This requirement is too broad or requires course approval from the concentrator's Director of Undergraduate Studies, so TigerPath does not check it.</i>\n\nThe specific courses to be taken by each student must form a coherent program of study. Whatever the individual concentration, each student's program must contain eight departmentals and the Junior Seminar.",
+              "req_list": [
+                {
+                  "name": "Classical Civilization",
+                  "max_counted": null,
+                  "min_needed": null,
+                  "explanation": "<i>Editor's Note: This requirement is too broad or requires course approval from the concentrator's Director of Undergraduate Studies, so TigerPath does not check it.</i>\n\nSix of these courses must focus in whole or in part on classical civilization or its influence. At least three of these courses must be taught in the Department of Classics (CLA, CLG, or LAT). Courses taken during the freshman and sophomore years beyond the prerequisite may count toward this requirement if they are appropriate to the student's overall program.",
+                  "no_req": null
+                },
+                {
+                  "name": "Comparative/Methodological Component",
+                  "max_counted": null,
+                  "min_needed": null,
+                  "explanation": "<i>Editor's Note: Concentrators must get departmental approval for many of these courses, and TigerPath does not verify this.</i>\n\nTwo courses must fulfill the comparative/methodological component of the track of study. The aim of this requirement is to introduce students to new perspectives and new tools of inquiry for exploring their chosen subject. There is no set list, since different courses will be appropriate to different interests. These courses must be preapproved by the DUS to count as part of the student's track.",

```

Remove the "many of", since all courses need approval.
```suggestion
                  "explanation": "<i>Editor's Note: Concentrators must get departmental approval for these courses, and TigerPath does not verify this.</i>\n\nTwo courses must fulfill the comparative/methodological component of the track of study. The aim of this requirement is to introduce students to new perspectives and new tools of inquiry for exploring their chosen subject. There is no set list, since different courses will be appropriate to different interests. These courses must be preapproved by the DUS to count as part of the student's track.",
```